examples/CMakeLists.txt

```diff
@@ -1,21 +1,21 @@
 add_definitions(-w)
 
 add_executable(simple-example simple/simple.cpp)
-target_link_libraries(simple-example PRIVATE photon_static)
+target_link_libraries(simple-example PRIVATE photon_shared)
 
```

Comment:
We'd better not change static libs to shared libs, because we usually need to copy example programs to other machines to run tests. An independent binary is convenient .